src/elements/content-sidebar/activity-feed/annotations/AnnotationActivityMenu.js

```diff
@@ -0,0 +1,81 @@
+// @flow
```

Comment:
Can we use TypeScript for these new components?

Reply:
I converted AnnotationActivityLink to TS, but AnnotationActivityMenu has been more challenging due to it re-using a couple widely used Flow components. Not sure if it should be in the scope of this PR to migrate those components.